main.py

```diff
@@ -18,6 +18,16 @@
     await ctx.send('pong')
 
 
+@bot.command()
+async def charada(ctx):
+    try:
+        r = requests.get(url="https://us-central1-kivson.cloudfunctions.net/charada-aleatoria",
+                         headers={'Accept': 'application/json'})
+        await ctx.send(r.json()['pergunta'])
+        await ctx.send(r.json()['resposta'])
+    except:
+        await ctx.send('Are you dumb?')
```

Comment:
It would be nice to change this to match the others, for random fallback answers.

Reply:
Maybe at some point we can wrap this whole flow, since we are pretty much repeating the try except in each command. Writing a decorator for this could be useful, I'll add that in the roadmap.